styles.css

```diff
@@ -320,3 +367,50 @@ header {
 .portfolio-details h4 {
   font-size: 24px;
 }
+
+.inputs {
+  display: flex;
+  flex-direction: column;
+}
+
+.contacr_form {
+  display: flex;
+  flex-direction: column;
+}
+
+.contact .form1 {
+  padding: 12px 67px 12px 0;
+  margin-top: 50px;
+  outline: none;
+}
+
+.contact .form2 {
+  padding: 12px 67px 12px 0;
+  margin-top: 20px;
+  outline: none;
+}
+
+.contact .text_form {
+  /* padding: 10px 67px 77px 0; */
```

Comment:
To be in line with the best practices in programming, I suggest that you remove commented pieces of code from you codebase. This makes you code clean and easy to read. 😉